osbs/conf.py

```diff
@@ -373,6 +360,14 @@ def get_reactor_config_map(self):
         return self._get_value("reactor_config_map", self.conf_section,
                                "reactor_config_map")
 
+    def get_reactor_config_map_scratch(self):
+        return self._get_value("reactor_config_map_scratch", self.conf_section,
+                               "reactor_config_map_scratch")
+
+    def get_pipeline_run_path(self):
+        return self._get_value("pipeline_run_path", self.conf_section,
+                               "pipeline_run_path")
```

Comment:
Where will the actual yaml template for a PipelineRun come from?

Reply:
prun template is in osbs-playbooks, we don't copy it yet anywhere in playbook (we have story for that), but this is full file path